### PR TITLE
feat(pci.storage): add encryption OMK tracking

### PR DESCRIPTION
**Pull request by darsene**
ref: MANAGER-13875

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/kms-omk`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-13875
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- ~Only FR translations have been updated~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~Breaking change is mentioned in relevant commits~

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/containers/container/container.html [hunk 1]
```diff
@@ -85,6 +85,7 @@
                         class="pl-1 fa fa-question-circle-o oui-color-p-500"
                         aria-hidden="true"
                         data-oui-tooltip="{{:: 'pci_projects_project_storages_containers_data_encryption_aes256_tooltip' | translate }}"
+                        data-ng-mouseover="$ctrl.trackEncryptionAction('TOOLTIP_AES256')"
```
Comment (JacquesLarique) [inline packages/manager/modules/pci/src/projects/project/storages/containers/container/container.html:88]: Isn't it "dangerous" to send tracking on a mouseover ?

**Review by JacquesLarique — APPROVED** [review]

**Review by chipp972 — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/containers/container/container.html (hunk 1 above)
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/containers/container/container.html:88]: Try stop propagating the event if the click is on tooltip.

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by JacquesLarique — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/containers/container/data-encryption/template.html [hunk 2]
```diff
@@ -2,17 +2,21 @@
...
                 data-ng-if="option.tooltip"
-                class="pl-1 fa fa-question-circle-o oui-color-p-500"
-                aria-hidden="true"
-                data-oui-tooltip="{{:: option.tooltip | translate }}"
-            ></i>
+                type="button"
+                class="pl-1 oui-color-p-500 oui-popover-button"
+                data-oui-popover="{{:: 'pci_projects_project_storages_containers_data_encryption_aes256_tooltip' | translate }}"
+                data-ng-click="$ctrl.trackEncryptionAction('TOOLTIP_' + option.value);$event.stopPropagation();"
+            ></button>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/containers/container/data-encryption/template.html:19]: **question:** Can you create a dedicated function?

Comment (darsene) [reply packages/manager/modules/pci/src/projects/project/storages/containers/container/data-encryption/template.html]: Fixed

**Review by anooparveti — APPROVED** [review]

**Pull request merged**